src/effects/effectchainslot.cpp

```diff
@@ -323,14 +329,6 @@ void EffectChainSlot::registerChannel(const ChannelHandleAndGroup& handle_group)
     m_channelStatusMapper.setMapping(pEnableControl, handle_group.name());
     connect(pEnableControl, SIGNAL(valueChanged(double)),
             &m_channelStatusMapper, SLOT(map()));
-
-    if (m_pEffectChain != nullptr) {
```

Comment:
I think we still need this. Or is m_pEffectChain always null? Than we need an debug assert.

Reply:
I don't think registering an input channel should be coupled with enabling it. Again, AFAIK this is working, and I'm hesitant to make further changes in this brittle system without a demonstrated need.

Reply:
OK